homeassistant/components/cast/config_flow.py

```diff
@@ -28,7 +33,22 @@ async def async_step_import(self, import_data=None):
         """Import data."""
         if self._async_current_entries():
             return self.async_abort(reason="single_instance_allowed")
-        data = {CONF_KNOWN_HOSTS: self._known_hosts}
+
+        media_player_config = self.hass.data[DOMAIN].get("media_player") or {}
```

Comment:
Why are we storing the configuration yaml in hass.data instead of passing it to the import step?

Reply:
Fixed in fa14da8